src/backend/expungeservice/charge_classifier.py

```diff
@@ -156,7 +156,7 @@ def _marijuana_violation(name, level):
     @staticmethod
     def _marijuana_eligible(section, name, birth_year, disposition):
         if section == "475860" or "marij" in name or "mj" in name.split():
-            if birth_year and disposition:
+            if birth_year and disposition.status != DispositionStatus.UNKNOWN:
```

Comment:
This is just a direct refactoring for the scope of this PR. But in a future PR, I do think that this should check for `disposition.convicted()` though.